shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/job/progress/yaml/YamlIncrementalTaskProgress.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.core.job.progress.yaml;
+
+import java.util.Map;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.shardingsphere.data.pipeline.api.task.progress.IncrementalTaskDelay;
+
+/**
+ * Yaml IncrementalTaskProgress.
+ */
+@Getter
+@Setter
+public final class YamlIncrementalTaskProgress {
+    
+    private Map<String, YamlIncrementalTaskProgressItem> dataSources;
+    
+    @Getter
+    @Setter
+    public static final class YamlIncrementalTaskProgressItem {
+        
+        private String position;
+        
+        private IncrementalTaskDelay delay;
+    }
+}
```

Comment:
Could we simplify `YamlIncrementalTaskProgress`?
`Map<String, YamlIncrementalTaskProgressItem>` just includes 1 key value pair. We could add `dataSourceName` and remove `YamlIncrementalTaskProgressItem`.

In fact, `JobProgress` doesn't represent whole job's progress, it represent sharding item's progress, one job might have several sharding items (sharding item is concept of ElasticJob).
Related code:
```
GovernanceRepositoryAPI.java
    JobProgress getJobProgress(String jobId, int shardingItem);

RuleAlteredJobAPI.java
    Map<Integer, JobProgress> getProgress(RuleAlteredJobConfiguration jobConfig);
```

Reply:
OK， job progress like ths？
```yaml
incremental:
    dataSourceName：ds_0
    delay:
      lastEventTimestamps: 0
      latestActiveTimeMillis: 0
    position: mysql-bin.000012#237010997
```

Reply:
Yes. You mentioned it in another issue before.